Dockerfile

```diff
@@ -59,6 +59,7 @@ RUN apt-get update && \
         unzip \
         openssh-server \
         dumb-init \
+        gosu \
```

Comment:
We can avoid adding `gosu` by splitting up the directory run from the docker entry point.